src/components/collapsible/Collapsible.tsx

```diff
@@ -117,6 +117,7 @@ class Collapsible extends React.PureComponent<CollapsibleProps, CollapsibleState
                         {...modifiedButtonProps}
                         className="collapsible-card-title"
                         onClick={this.toggleVisibility}
+                        aria-expanded={isOpen ? 'true' : 'false'}
```

Comment:
It looks like `aria-expanded` expects a boolean. Can use `aria-expanded={isOpen}` here? Also, can we keep the props alphabetized?